### DISCUSSION TITLE
Impact of different round-off errors between v1.1 and v1.2 emission factors

### DISCUSSION THREAD

**Discussion by damienlieber** [USEPA/supply-chain-factors#51] [General]
Hi all,

I noticed that some emission factors increased between v1.1 and v1.2, particularly in services. For example, "Advertising and public relations" emissions increased. I would have expected these emission factors to decrease, in particular due to the decarbonization of the electric grid between 2016 and 2019. 

I noticed that at least some of that increase is explained by different rounding methods in the output Excels used in corporate carbon accounting. In v1.1, values for CO2, CH4, N2O, other GHGs are standard numbers rounded to the third decimal. In v1.2, they are scientific numbers rounded to the second decimal. 

Going back to the example of "Advertising and public relations", Methane = 0 kg/USD_2018 and Nitrous oxide = 0 kg/USD_2018 in v1.1. These values were used by carbon accountants to estimate the total kgCO2e/USD_2018 emission factor for advertising and public relations, which resulted in no contribution from Methane and Nitrous oxide. We now see in v1.2 that Methane = 3.42E-4 kg/USD = 0.009 kgCO2e/USD and Nitrous oxide = 1.82E-5 kg/USD = 0.005 kgCO2e/USD. Together, they represent ~12% of the total 0.129 kgCO2e/USD_2021 = 0.119 in 2022 USD. This is a 16% increase over the 0.103 kgCO2e/USD_2022 in v1.1.

Could you please confirm the above and consider communicating on the difference before the corporate accounting season for 2022 starts?

Thank you!

Damien

**Comment by bl-young** [comment]
You are correct that in the published files the precision has increased in the most recent release (v1.2) compared to earlier versions.

[This document](https://pasteur.epa.gov/uploads/10.23719/1528686/documents/Aboutthe2019v1.2SupplyChainGHGEmissionFactors.pdf) describes some of the changes between v1.1 and v1.2 (which you can find attached to the [dataset](https://edg.epa.gov/metadata/catalog/search/resource/details.page?uuid=https://doi.org/10.23719/1528686)).
